odoo/addons/base/models/res_company.py

```diff
@@ -326,6 +327,12 @@ def _unlink_if_company_has_no_children(self):
     def write(self, values):
```

Comment:
What about `create`?

Reply:
Do we need it? Because most of the time I guess write is supposed to be called, create does get calls when you create you branch and save it but in cases of branches we are restricting them when the currency doesn't match

If you want I can still add it if we have particular use case for that